README.md

```diff
@@ -46,6 +46,13 @@ uv run fastapi dev stapi-fastapi/tests/application.py
 
 The app should be accessible at `http://localhost:8000`.
 
+### stapi-client
+
+The `stapi-client` package is a Python client (CLI tool) focused on interfacing with the STAPI specification, specifically with a STAPI server. This is a work in progress.
+
+For more information, see the [README](pystapi-client/README.md).
+
+
```

Comment:
Can we move this down into the `## Packages` section? We could move the `stapi-fastapi` section as well... 🙇🏼